aten/src/ATen/native/README.md

```diff
@@ -170,6 +170,27 @@ Note that argument type modifiers such as defaults and optional are not currentl
 
 The declarations also support the following attributes:
 
+**Overloads.** You can register multiple functions with the same name and different
+function signatures if you give them unique overload names. An overload name
+is specified after the function name, separated by a dot.
+
+Overload names do not have to be globally unique, but must be unique in the set
+of all overloads for the same function. Please try to make overload names
+semantically meaningful. An overload name that just enumerates the argument
+types isn't helpful. Actually it is hurtful, because overload names cannot be
+changed for backwards compatibility reasons, even if for some reason function
+argument types change.
```

Comment:
wait, I can change the argument types but I can't change the overload name?  If that's the case this needs to be explained, i.e. what is allowed to change while still being the same overload?  (compatible types from python and new default parameters?)